ckanext/switzerland/harvester/base_sbb_harvester.py

```diff
@@ -995,8 +996,10 @@ def _get_ordered_resources(self, package):
 
         return ordered_resources, unmatched_resources
 
-    def finalize(self, harvest_object, harvest_object_data):
-        context = {"model": model, "session": Session, "user": self._get_user_name()}
+    def finalize(self, harvest_object, harvest_object_data):  # noqa: C901
+        # TODO: Simplify this method.
```

Comment:
I haaaate doing this, but this code needs refactoring anyway and I didn't want to do that in this same PR, which is already a bit complex. :-/